src/Orleans.Streaming.AzureStorage/Orleans.Streaming.AzureStorage.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Streaming.AzureStorage</PackageId>
+    <Title>Microsoft Orleans Streaming Azure Table Storage</Title>
+    <Description>Streaming storage providers for Azure Storage of Microsoft Orleans.</Description>
+    <PackageTags>$(PackageTags) Azure Table Blob Storage</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>Orleans.Streaming.AzureStorage</AssemblyName>
```

Comment:
>Orleans.Streaming.AzureStorage [](start = 18, length = 30)

I think Orleans.Streaming.AzureQueue would be more precise.

Reply:
I don't know about this one, I would put Queues in the description, but AzureStorage is the branding we are using elsewhere for package ids.